src/XenobladeChroniclesX/Enhancements/AntiAliasing/rules.txt

```diff
@@ -8,9 +8,9 @@ version = 6
 [Default]
 $Preset:int = 0
 
-$subPix:float = 0.75
-$edgeThreshold:float = 0.166
-$edgeThresholdMin:float = 0.0833
+$subPix:double = 0.75
+$edgeThreshold:double = 0.166
+$edgeThresholdMin:double = 0.0833
```

Comment:
I have tested :double, the mod works just the same, there is a noticable difference between native (mod off), fxaa, and aa disable.
so working exactly as intended